gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -114,32 +148,99 @@ def call_quit(self, _):
         """Process Quit command - quit the program"""
         return True
 
-    def call_cg(self, _):
-        if pycoingecko_controller.menu():
-            return True
-        print("")
+    def call_load(self, other_args):
+        """Process load command"""
+        try:
+            self.current_coin, self.source = load(
+                coin=self.current_coin, other_args=other_args
+            )
+        except TypeError:
+            print("Couldn't load data\n")
+
+    def call_chart(self, other_args):
+        """Process chart command"""
+        if self.current_coin:
+            getattr(DD_VIEWS_MAPPING[self.source], "chart")(
```

Comment:
This chart could use the mplfinance lib as it looks nicer :)

Feel free to not take some of these actions on this PR, and work on these at a later stage if needed

Reply:
In coingecko, there are only "price" column, so we don't have there Open, High, Low, Close, Volume. Mplfinance is amazing, but one constraint is to have OHLCV data, so for coingecko it's not aplicable.

Reply:
This makes sensee, thanks for the explanation!

What do you think about having another default dataframe data source? So that we can have mplfinance?